CMakeLists.txt

```diff
@@ -16,6 +16,11 @@ if (SKBUILD)
   set(Python_EXECUTABLE "${PYTHON_EXECUTABLE}")
   set(Python_INCLUDE_DIR "${PYTHON_INCLUDE_DIR}")
   set(Python_LIBRARIES "${PYTHON_LIBRARY}")
+else()
+  # Find any Python version (this should be narrowed to the one with nanobind)
+  find_package(PythonInterp)
```

Comment:
Can these three lines be deleted? There is
```
find_package(Python COMPONENTS Interpreter Development.Module REQUIRED)
```
just below which subsumes this.

Reply:
You'd think, but when I remove those, I get this in VS Code:
```
[cmake] CMake Error at C:/Program Files/CMake/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
[cmake]   Could NOT find Python (missing: Python_EXECUTABLE Python_LIBRARIES
[cmake]   Python_INCLUDE_DIRS Interpreter Development.Module)
```

Without `find_package(PythonInterp)`, I get:
```
[cmake] CMake Error at CMakeLists.txt:34 (find_package):
[cmake]   Could not find a package configuration file provided by "nanobind" with any
[cmake]   of the following names:
[cmake] 
[cmake]     nanobindConfig.cmake
[cmake]     nanobind-config.cmake
[cmake] 
[cmake]   Add the installation prefix of "nanobind" to CMAKE_PREFIX_PATH or set
[cmake]   "nanobind_DIR" to a directory containing one of the above files.  If
[cmake]   "nanobind" provides a separate development package or SDK, be sure it has
[cmake]   been installed.
```

However it seems like `find_package(PythonLibs)` is unnecessary, so I'll remove that. :+1:

Reply:
Can this issue be decoupled from VSCode? Does the issue happen when you run cmake normally on the windows command line? (e.g. for a pure visual studio build)

Reply:
If not, is there a cmake `if` declaration that one can use to realize that CMake is being somehow invoked in "Visual Studio Code" mode?

Reply:
I get identical errors in Visual Studio proper and the CMake GUI (configured to the Visual Studio Generators).

Yes; I'll add the `if(MSVC)` condition now...